pages/dkp/kommander/2.2/install/small-environment/index.md

```diff
@@ -30,60 +32,69 @@ In this YAML file you can find commented lines, that correspond to all platform
 
 For example, if you want to test Grafana to allow monitoring, remove the pound sign preceding `grafana-logging: null`. Since Grafana depends on Loki, you also have to remove the pound preceding `grafana-loki: null` and `minio-operator: null`. Note that depending on the size of your cluster, enabling several platform applications could exhaust your cluster’s resources.
 
-1.  Edit your `install.yaml` file to match the following example:
+1.  Initialize your Kommander installation and name it `kommander.minimal`:
```

Comment:
GH isn't giving me the option to suggest edits for some reason :( but this needs to be fixed to be `kommander_minimal.yaml`